ledger-core/virtual/execute/execute.go

```diff
@@ -333,6 +336,14 @@ func negotiateIsolation(methodIsolation, callIsolation contract.MethodIsolation)
 	return res, nil
 }
 
+func checkAllowedIsolation(isolation contract.MethodIsolation) {
```

Comment:
I don't see a single reason why you've extracted that check to separate function, but it's not important.

Reply:
inlined the function